.jshintrc

```diff
@@ -20,6 +20,9 @@
   "worker": true,
   "globals": {
     "JSON": false,
-    "URLSearchParams": false
+    "URLSearchParams": false,
+    "define": true,
+    "module": true,
+    "global": true
```

Comment:
Isn't there a jshint "environment" declaration akin to `"worker": true` that makes it assume that these globals are available; possibly `"umd": true` or `"amd": true`?